library/styleguideComponents/UniversalSearch/PostItemSearchResult.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
I know we have been inconsistent in the library code, but we can't follow with this inconsistency, I think we should start all files with lower case